gravitee-apim-console-webui/src/management/settings/shared-policy-groups/shared-policy-groups.component.ts

```diff
@@ -130,6 +130,7 @@ export class SharedPolicyGroupsComponent implements OnInit {
             phase: toReadableFlowPhase(sharedPolicyGroup.phase),
             updatedAt: sharedPolicyGroup.updatedAt,
             deployedAt: sharedPolicyGroup.deployedAt,
+            isReadOnly: sharedPolicyGroup.originContext?.origin === 'KUBERNETES',
```

Comment:
To be read again. What do you think about combining isReadonly here ? 
Move here the condition L108. and use only this isReadOnly

Reply:
see ☝️ as this flag is really about showing the kubernetes icon, I made it more specific and it's now `isKubernetesOrigin`